packages/cf-component-icon/src/Icon.js

```diff
@@ -76,11 +288,18 @@ Icon.propTypes = {
     'widen',
     'wrench'
   ]).isRequired,
-  size: PropTypes.oneOf(['2x', '3x', '4x', 'large', 'xlarge']),
```

Comment:
FWIW, we're using `size="large"` in the www-next codebase, but it's only in one place.